src/playback/internal/playbackuiactions.cpp

```diff
@@ -125,6 +125,14 @@ const UiActionList PlaybackUiActions::m_settingsActions = {
              IconCode::Code::CHORD_SYMBOL,
              Checkable::Yes
              ),
+    UiAction("toggle-hear-playback-when-editing",
+             mu::context::UiCtxAny,
+             mu::context::CTX_ANY,
+             TranslatableString("action", "Hear playback when editing"),
+             TranslatableString("action", "Hear playback when editing"),
```

Comment:
@avvvvve do these names look good now?
The first one is how it appears in the UI; the second one is how it appears in Preferences > Shortcuts.

Reply:
@pacebes @cbjeukendrup 
The menu should say "Hear playback when editing" (as it does now)
The preferences action should say "Toggle hear playback when editing" (currently incorrect)

Also, let's use the regular speaker icon instead of the pencil in both places (U+EF4E). Thought on it and we want to reserve the pencil specifically for note input mode.

Reply:
Ok.  I have just upload the changes